### PR TITLE
ruby: remove 1.9 (WIP)

### PR DESCRIPTION
**Pull request by vrthra**
###### Motivation for this change

Ruby 1.9 is EOLd by upstream
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
cc @zimbatm

File: pkgs/development/libraries/sdformat/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchurl, cmake, boost, ruby_1_9, ignition, tinyxml
+{ stdenv, fetchurl, cmake, boost, ruby_2_2, ignition, tinyxml
   , name ? "sdformat-${version}"
   , version ? "4.0.0" # versions known to work with this expression include 3.7.0
   , srchash-sha256 ? "b0f94bb40b0d83e35ff250a7916fdfd6df5cdc1e60c47bc53dd2da5e2378163e"
   , ...
   }:
 
 let 
-  ruby = ruby_1_9;
```
Comment (zimbatm) [inline pkgs/development/libraries/sdformat/default.nix]: I bet there is a reason why it had been set to ruby 1.9. Did you test this library?

**Comment by zimbatm** [comment]
Good idea. :+1:

File: pkgs/development/libraries/sdformat/default.nix (hunk 1 above)
Comment (zimbatm) [inline pkgs/development/libraries/sdformat/default.nix]: Maybe @pxc knows. He's the implicit maintainer as the gazebo package is the only one depending on this lib.

Comment (vrthra) [inline pkgs/development/libraries/sdformat/default.nix]: No, I only checked that it compiled fine. I will check this and get back.

**Comment by fpletz** [comment]
Any news here?

**Comment by vrthra** [comment]
@fpletz I am currently on a slow and sporadic internet connection (this month and for the next month too). Hence I have been unable to move forward on this. I changed the title to WIP, and can close it for now and reopen it later if that is better.

**Comment by matthiasbeyer** [comment]
Any progress?

**Comment by zimbatm** [comment]
Squashed and merged into master. sdformat is marked as broken until @pxc fixes it.

**Comment by pxc** [comment]
Not sure which user you want to notify @zimbatm but it's not me ;-)

**Pull request closed without merging**